lib/crawler_web/templates/home/index.html.heex

```diff
@@ -1,7 +1,64 @@
-<div class="mt-3">
+<div class="mt-3" aria-label="breadcrumb">
   <div class="container">
-    <h1>
+    <h1 aria-current="page">
       <%= gettext("Home") %>
     </h1>
+
+    <h4>
+      <%= gettext("Please choose a CSV file:") %>
+    </h4>
+
+    <%= form_for @changeset, Routes.keyword_path(@conn, :create), [method: "post" , class: "row pt-2" , multipart:
+      true], fn f-> %>
+      <div class="col-3">
+        <%= file_input(f, :file, class: "form-control", required: true, accept: 'text/csv') %>
+      </div>
+      <div class="col-3">
+        <%= submit(gettext("Upload"), class: "btn btn-primary mb-3") %>
+      </div>
+    <% end %>
+
+    <%= if Enum.empty?(@keywords) do %>
+      <p>
+        <%= gettext("You don't have keywords yet, please upload a CSV file.") %>
+      </p>
+    <% else %>
+      <div class="table-responsive">
+        <table class="table table-hover">
+          <thead>
+            <tr>
+              <th scope="col">#</th>
+              <th scope="col">
+                <%= gettext("Keyword") %>
+              </th>
+              <th scope="col">
+                <%= gettext("Status") %>
+              </th>
+              <th scope="col">
+                <%= gettext("Updated") %> (UTC)
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <%= for {keyword, i} <- Enum.with_index(@keywords, 1) do %>
+              <tr>
+                <td>
+                  <%= i %>
```

Comment:
instead using index, can we use `keyword.id`

Reply:
Updated in [1d816b2](https://github.com/vnntsu/elixir-ic/pull/36/commits/1d816b234f96b49e84b553a585e8119e4990ed31)